core-rust/core-api-server/src/core_api/conversions/addressing.rs

```diff
@@ -271,6 +273,18 @@ fn to_mapped_substate_id(substate_id: SubstateId) -> Result<MappedSubstateId, Ma
             };
             (EntityType::Component, substate_type_key)
         }
+        SubstateId(RENodeId::AccessController(_), offset) => {
+            let substate_type_key = match offset {
+                SubstateOffset::AccessController(offset) => match offset {
+                    AccessControllerOffset::AccessController => (
+                        SubstateType::AccessController,
+                        SubstateKeyType::AccessController,
+                    ),
+                },
+                _ => return Err(unknown_substate_error("AccessController", &substate_id)),
```

Comment:
I imagine we're missing at least AccessRulesChain and Metadata here?

Reply:
Actually looking at https://github.com/radixdlt/radixdlt-scrypto/blob/release/betanet-v2/radix-engine/src/engine/node.rs it looks like it only supports `AccessRulesChain`. But perhaps we should also map out Metadata anyway?

Reply:
Cool, yeah good catch, will add these.